artifactory/cli_test.go

```diff
@@ -30,3 +41,135 @@ func TestValidateGoNativeCommand(t *testing.T) {
 		})
 	}
 }
+
+func TestPrepareSearchDownloadDeleteCommands(t *testing.T) {
+	tests := []struct {
+		name            string
+		args            []string
+		flags           []string
+		expectedPattern string
+		expectedBuild   string
+		expectedBundle  string
```

Comment:
The expectedBundle property seems to be unused. Same foes for the other new test functions.